maskrcnn_benchmark/config/paths_catalog.py

```diff
@@ -6,38 +6,86 @@
 
 class DatasetCatalog(object):
```

Comment:
in the top of this file and `defaults.py` there is a `100644 → 100755`. I believe this is a change in permission (via chmod). Can you verify and change back to what it was?

Reply:
I've changed back. Thanks for the careful review!